system/jlib/jfile.cpp

```diff
@@ -2055,11 +2059,19 @@ CFileIO::CFileIO(HANDLE handle, IFOmode _openmode, IFSHmode _sharemode, IFEflags
     sharemode = _sharemode;
     openmode = _openmode;
     extraFlags = _extraFlags;
+
+    // leave for compatibility
     if (extraFlags & IFEnocache)
         if (!isPCFlushAllowed())
             extraFlags = static_cast<IFEflags>(extraFlags & ~IFEnocache);
```

Comment:
could this be moved/combined into a expertDisableIFileFlagsMask in the updateFunc?

Reply:
I don't think so because we don't know extraFlags at this point and if we set expertDisableFileFlagsMask here it is too early.